configs/ddppo_pointnav.yaml

```diff
@@ -6,7 +6,7 @@ TORCH_GPU_ID: 0
 VIDEO_OPTION: []
 TENSORBOARD_DIR: "tb"
 VIDEO_DIR: "video_dir"
-TEST_EPISODE_COUNT: -1
+TEST_EPISODE_COUNT: 994
```

Comment:
Is this change intentional?

Reply:
Yeah, because I was worried it might cause issues with people still on habitat-stable. Should I change it back?

Reply:
@erikwijmans -- what do you suggest?

Reply:
Habitat stable isn't useable for the challenge as is, so I'm not worried about that.  I think the challenge 2020 tag is usable however (which is before the PR needed for -1).  994 is fine as it should be clear that this config is the pointnav dataset.  The reason to do -1 instead of 994 is if we think people will use this config for other datasets and not understand that they need to change the 994.